README.md

```diff
@@ -26,54 +26,105 @@ A swagger-ui page is available at /swagger-ui/ on any running instance. For exis
 ### Framework
 This project uses the [Flask](https://flask.palletsprojects.com/en/1.1.x/) Python web framework.
 
-## Developing
+
+## Managing dependencies
+
+We use [Poetry](https://python-poetry.org/docs/) to manage our dependencies. From their website: 
+
+> Poetry is a tool for dependency management and packaging in Python. It allows you to declare the libraries your project depends on and it will manage (install/update) them for you. Poetry offers a lockfile to ensure repeatable installs, and can build your project for distribution.
+
+
+Install [Poetry](https://python-poetry.org/docs/)
+
+```sh
+curl -sSL https://install.python-poetry.org | python3 -
+```
+
+If you need to change any dependency versions:
+- update the pyproject.toml file
+- run the following to update the lock file
+  
+```sh
+poetry lock
+```
+
 Install dependencies
+```sh
+poetry install
+```
+
+Update dependencies
+```sh
+poetry update
+```
+
+Export to requirements.txt. Google App Engine uses this for deployment.
```

Comment:
Oh boy okay, so Google App Engine won't take a poetry.lock file and we need a requirements.txt? :')
The only potential hitch, is that requirements.txt are not platform agnostic like poetry is. So if you install things on your local mac for instance, it might not translate well to an ubuntu instance. I think it is fine for now, but maybe leaving a warning in the readme would be good.

Reply:
yeahhhh not ideal for sure, I'll add more context in the readme

Reply:
We use google app engine anymore though, its through helm/argo like the rest of our services https://argocd.dsp-devops-prod.broadinstitute.org/applications/calhoun-dev?resource=